include/logInFunctions.php

```diff
@@ -0,0 +1,71 @@
+
+<script src="/include/jquery.js"></script>
```

Comment:
script files shouldn't be directly in include files like this. For the most part, nothing should ever go in an include file except PHP functions. What this does is it echoes all this code to the page immediately when it runs, so it's not in the actual `<html>` tag that is supposed to wrap the whole page.

Instead, take all this js code and put it in your page header function.